Composer/packages/client/setupTests.ts

```diff
@@ -1,5 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+/// <reference types="jest" />
+
 // for tests using Electron IPC to talk to main process
 (window as any).ipcRenderer = { on: jest.fn() };
+
+jest.mock('./src/utils/httpUtil');
```

Comment:
One thing to consider here is  we have seen dev's  forget to mock axios operations and the test tries to make an actual HTTP call. I wonder if it makes sense to provide a default mock implementation to axios in setupTest to avoid suchscenarios.

Reply:
I'll add one and run the test suite